### ISSUE TITLE
Retrolab vs classic notebook - occasional weird highlighting in tables

### ISSUE THREAD

**Issue by davidwagner** [berkeley-dsep-infra/hubs#2994]
In Data 8 lec10 demos cells 27-31, when displayed with Retrolab, for some reason Retrolab weirdly decides to highlight one of the outputs with a gray border but not any others.  This doesn't seem to occur with the classic notebook.

classic notebook: http://data8.datahub.berkeley.edu/hub/user-redirect/git-sync?repo=https://github.com/data-8/materials-fa21&subPath=lec/lec10.ipynb

![lec10cell27-31classic](https://user-images.githubusercontent.com/3349841/141423820-88186d1a-ea63-4c01-a41a-927b72c1d6dc.png)

Retrolab: https://data8.datahub.berkeley.edu/hub/user-redirect/git-pull?repo=https%3A%2F%2Fgithub.com%2Fdata-8%2Fmaterials-fa21&branch=master&urlpath=retro%2Ftree%2Fmaterials-fa21%2Flec%2Flec10.ipynb

![lec10cells27-31retro](https://user-images.githubusercontent.com/3349841/141423830-031337a2-62cd-4af8-9256-8a29ddb348cc.png)

I didn't have keyboard focus on that particular table (I can click around and that one output continues to have an extra gray border, while the other outputs do not).  I don't understand why that one output was framed.

Obviously this is super-minor, but it's a display glitch that doesn't occur with the classic notebook.

**Comment by balajialg** [comment]
Thanks, @davidwagner for raising the varied requests related to Retrolab. I will close these issues as soon as the corresponding changes are made upstream.

@yuvipanda @jtpio Who would be the right person to escalate such retro lab-related issues in the future?

**Comment by ryanlovett** [comment]
@balajialg You can probably search for existing issues or create new ones at https://github.com/jupyterlab/retrolab/issues.

**Comment by balajialg** [comment]
@ryanlovett Thanks a lot! Appreciate it

**Comment by jtpio** [comment]
@balajialg yeah feel free to open new issues on the retrolab repo.

You can use this Binder (or something similar) to create examples that are easy to reproduce: https://gist.github.com/jtpio/4a6a34da66b32249e28df718ec30e4d7

(since we don't have access to the data8 deployment)

**Comment by jtpio** [comment]
> Retrolab: https://data8.datahub.berkeley.edu/hub/user-redirect/git-pull?repo=https%3A%2F%2Fgithub.com%2Fdata-8%2Fmaterials-fa21&branch=master&urlpath=retro%2Ftree%2Fmaterials-fa21%2Flec%2Flec10.ipynb

@davidwagner I just tried using this link and it looks related to the cell having `scrolled` set to true in its metadata. This can be checked in JupyterLab by opening the notebook with the Editor:

![image](https://user-images.githubusercontent.com/591645/142072425-65d6df77-2315-4706-afac-348192f5576b.png)

Right click and "Disable Scrolling for Outputs" removes it. 

Could it be that some of the notebooks pulled from the materials repo already have `scrolled` set to true in some of the cells?

**Comment by yuvipanda** [comment]
Also looks like something that will be fixed with https://github.com/jupyterlab/retrolab/pull/288

**Comment by yuvipanda** [comment]
This is still present with the latest version of retrolab

<img width="1402" alt="image" src="https://user-images.githubusercontent.com/30430/147960502-0863df91-f968-453c-92a3-614245c30213.png">

**Comment by jtpio** [comment]
> I just tried using this link and it looks related to the cell having scrolled set to true in its metadata.

Probably for this one removing the `scrolled` flag from the notebook metadata should be enough?

**Comment by fperez** [comment]
While it may look a little odd, if anything this behavior is actually perhaps a little better than in classic, in that it makes it visually explicit to know which cells have their scroll tag on. It's worth remembering that the scrolling can be toggled from the right-click menu, it doesn't require editing the raw JSON:

![image](https://user-images.githubusercontent.com/57394/150213502-86220bfe-fffb-49bf-b832-46d633ef421c.png)

We can argue about how it's displayed, but honestly I think this one is actually a feature, not a bug, and I'd close it as such :)

**Comment by jtpio** [comment]
Yes JupyterLab (and hence RetroLab) provides commands to control the scrolled state of a cell.

I just tried on stock `retrolab==0.3.16` (latest release) with this gist: https://gist.github.com/jtpio/4a6a34da66b32249e28df718ec30e4d7

And it seems to be working fine. This is what it looks like when `scrolled` is set to `true` in the cell metadata:

https://user-images.githubusercontent.com/591645/150307462-59ff4ca3-6ff2-4810-b589-de2af22bd5bb.mp4

So as mentioned above probably some of the material notebooks have `scrolled` set to `true` in their metadata?
